config/locales/en/admin.yml

```diff
@@ -1638,6 +1638,7 @@ en:
       index:
         create: Create tenant
       new:
+        admin_information: 'When you create a tenant, your current user <strong>"%{username}"</strong> with the email <strong>"%{email}"</strong> will be copied into the database of the new tenant and will be automatically granted administration permissions. Note that these will be user accounts that are stored in completely separate databases, so if you change the password of your user in one tenant, your user account passwords you have in other tenants will remain intact.'
```

Comment:
The end of the sentence sounds a bit funny :thinking:. Maybe we could end it with "your accounts in other tenants will remain intact.", which is similar to the text we use in Spanish :thinking:. What do you think?

Reply:
Yes, indeed 😆 . I got your suggestion and tried to simplify the last sentence a bit more.